src/activate_linux.c

```diff
@@ -268,7 +290,16 @@ int main(int argc, char *argv[]) {
     overlay_width *= scale;
     verbose_printf("Scaled width:  %d px\n", overlay_width);
 
+    if(display_screen == 0)
+        display_screen = 0xFFFFFFFF;
+
     for (int i = 0; i < num_entries; i++) {
+        if(!(display_screen & 1 << i))
```

Comment:
Such loop also exists a bit down in code. Please, add exclusion code there also (without printf, I think. There it will be useless).
https://github.com/MrGlockenspiel/activate-linux/blob/8a4d6b476f18d135a6052c5e64268e6b53320ba0/src/activate_linux.c#L331

Reply:
Yep, I realised that a bit later, it has been fixed in commit f7fc5fb0